.github/workflows/python-tests.yml

```diff
@@ -27,7 +27,14 @@ jobs:
     - name: Lint with flake8
       run: |
         pip install flake8
-        flake8 . --count --max-complexity=10 --statistics
+        flake8 hw_diag/. --count --max-complexity=10 --statistics
+    - name: Move migrations to expected location
+      run: |
+        mkdir /opt/migrations
+        cp -r migrations /opt/migrations/.
+        cp -r alembic.ini /opt/migrations/.
+        sudo mkdir /var/data
+        sudo chmod 777 /var/data
     - name: Unit tests
       run: |
         pip install pytest
```

Comment:
Shall we add pytest-cov to test-requirememts and add

--cov=hw_diag --cov=bigquery --cov-fail-under=70

Reply:
Generally I would agree, but I think it's out of scope of this PR. Let's raise this in a separate PR if there is a policy change regarding coverage.

Reply:
I mean, we have that on other repos such as config https://github.com/NebraLtd/hm-config/blob/c580b108ecacedae69b3b0c2954704646d2118a0/.github/workflows/python-tests.yml#L22

But yeah can put in a follow up ticket if that's easier

Reply:
Follow up ticket #445